scripts/src/javatests/org/oppia/android/scripts/coverage/CoverageRunnerTest.kt

```diff
@@ -40,89 +82,112 @@ class CoverageRunnerTest {
   }
 
   @Test
-  fun testRunWithCoverageAsync_emptyDirectory_throwsException() {
+  fun testRetrieveCoverageDataForTestTarget_emptyDirectory_throwsException() {
     val exception = assertThrows<IllegalStateException>() {
-      runBlocking {
-        coverageRunner.runWithCoverageAsync(bazelTestTarget).await()
-      }
+      coverageRunner.retrieveCoverageDataForTestTarget(bazelTestTarget)
     }
 
     assertThat(exception).hasMessageThat().contains("not invoked from within a workspace")
   }
 
   @Test
-  fun testRunWithCoverageAsync_invalidTestTarget_throwsException() {
+  fun testRetrieveCoverageDataForTestTarget_invalidTestTarget_throwsException() {
     testBazelWorkspace.initEmptyWorkspace()
 
     val exception = assertThrows<IllegalStateException>() {
-      runBlocking {
-        coverageRunner.runWithCoverageAsync(bazelTestTarget).await()
-      }
+      coverageRunner.retrieveCoverageDataForTestTarget(bazelTestTarget)
     }
 
     assertThat(exception).hasMessageThat().contains("Expected non-zero exit code")
     assertThat(exception).hasMessageThat().contains("no such package")
   }
 
   @Test
-  fun testRunWithCoverageAsync_validSampleTestTarget_returnsCoverageData() {
+  fun testRetrieveCoverageDataForTestTarget_coverageRetrievalFailed_throwsException() {
```

Comment:
Ok, so since the 1st part already mentions it as `RetrieveCoverageDataForTestTarget` I updated the names for the coverage missing cases as such:
- testRetrieveCoverageDataForTestTarget_coverageRetrievalFailed_throwsException -> **testRetrieveCoverageDataForTestTarget_withIncorrectPackageStructure_throwsException**
- testRetrieveCoverageDataForTestTarget_coverageDataMissing_throwsException -> **testRetrieveCoverageDataForTestTarget_withNoDepsToSourceFile_throwsException**

Does this accurately represent the cases?

I considered naming them as: 
- testRetrieveCoverageDataForTestTarget_retrievalFailedWithIncorrectPackageStructure_throwsException
- testRetrieveCoverageDataForTestTarget_missingCoverageDataWithNoDepsToSourceFile_throwsException

but they just felt a bit too long.

Reply:
I think the new names are quite clear--thanks!